hbw/app/javascript/packs/hbw/widget.js.jsx

```diff
@@ -61,7 +67,18 @@ modulejs.define(
       };
 
       render = () => {
-        this.renderPortals();
+        if (this.appReady) {
+          this.renderPortals();
+        } else {
+          const checkReady = () => {
+            if (this.appReady) {
+              this.renderPortals();
+            } else {
+              setTimeout(checkReady, 50);
+            }
+          };
+          checkReady();
+        }
```

Comment:
you can bind the needed function call to the `'hbw:app-rendered'` event directly, can't you?